### PR TITLE
move to kernel 6.8.1

### PR DESCRIPTION
**Pull request by xleonzhao**
fixed couple of function calls/args to be able to compile on kernel 6.8.1, tested ok.

### CODE REVIEW

File: krping.c [hunk 1]
```diff
@@ -1805,7 +1813,7 @@ static void krping_fr_test(struct krping_cb *cb)
 	inv.send_flags = IB_SEND_SIGNALED;
 	
 	DEBUG_LOG("fr_test: stag index 0x%x plen %u size %u depth %u\n", mr->rkey >> 8, plen, cb->size, cb->txdepth);
-	start = get_seconds();
+	start = ktime_get_seconds();
```
Comment (larrystevenwise) [inline krping.c]: since you call ktime_get_seconds() here, can you now remove get_seconds()?

Comment (xleonzhao) [reply krping.c]: Actually, I added a new function named `get_seconds()` which calls `ktime_get_seconds()`, just trying to minimize the changes to your original code.

File: krping.c [hunk 2]
```diff
@@ -59,7 +59,7 @@
 
 #define PFX "krping: "
 
-static int debug = 0;
+static int debug = 1;
```
Comment (larrystevenwise) [inline krping.c]: Please remove this; no need to default debug to on.

File: krping.c [hunk 3]
```diff
@@ -1823,7 +1831,8 @@ static void krping_fr_test(struct krping_cb *cb)
 			fr.key = mr->rkey;
 			inv.ex.invalidate_rkey = mr->rkey;
 
-			size = prandom_u32() % cb->size;
+			// size = prandom_u32() % cb->size;
```
Comment (larrystevenwise) [inline krping.c]: Please remove the commented-out old code.  no need to keep it.  We have git to see the old code. :)

File: krping.c [hunk 4]
```diff
@@ -1776,6 +1783,7 @@ static void krping_fr_test(struct krping_cb *cb)
 	int size = cb->size;
 	int plen = (((size - 1) & PAGE_MASK) + PAGE_SIZE) >> PAGE_SHIFT;
 	unsigned long start;
+	// time64_t start;
```
Comment (larrystevenwise) [inline krping.c]: Please remove this instead of commenting it out.

**Review by larrystevenwise — CHANGES_REQUESTED** [review]
i'll merge after you please fix the few nit comments i just added.  Thanks!

File: krping.c (hunk 2 above)
Comment (xleonzhao) [reply krping.c]: Ops, forgot changing it back..

**Comment by xleonzhao** [comment]
done.

**Comment by larrystevenwise** [comment]
Merged.  Thanks!

**Pull request merged**